src/composables/useSelectOrganization.ts

```diff
@@ -0,0 +1,85 @@
+import { computed } from 'vue';
+
+// stores
+import { useRegisterChallengeStore } from '../stores/registerChallenge';
+
+// types
+import type { Organization } from '../components/types/Organization';
+import type {
+  FormCompanyAddressFields,
+  FormSelectTableOption,
+} from 'src/components/types/Form';
+
+export const useSelectOrganization = (organizations: Organization[]) => {
```

Comment:
`useSelectedOrganization` no? Same for file name.

Reply:
Renamed composable.